src/aqua_diagnostics/seaice/seaice.py

```diff
@@ -0,0 +1,254 @@
+        as a new user may not know where the file is located."""
+        pathloc = f"Regions file location path: {ConfigPath().get_config_dir()}"
+        print(pathloc)
+
+        return self.regions_definition
+
+    @staticmethod
+    def set_seaice_update_attrs(seaice_computed, method: str, region: str):
+        """Set attributes for seaice_computed."""
+
+        # set attributes: 'method','unit'   
+        units_dict = {"extent": "million km^2", 
+                      "volume": "million km^3"}
+
+        if method not in units_dict:
+            raise NoDataError("Variable not found in dataset")
+        else:
+            seaice_computed.attrs["units"] = units_dict.get(method)
+
+        seaice_computed.attrs["long_name"] = f"Sea ice {method} integrated over {region} region"
+        seaice_computed.attrs["standard_name"] = f"{region} sea ice {method}"
+        seaice_computed.name = f"sea_ice_{method}_{region.replace(' ', '_').lower()}"
+
+        log_history(seaice_computed, f"Method used for seaice computation: {method}")
+        log_history(seaice_computed, f"Region selected for seaice computation: {region}")
+
+        return seaice_computed
+
+    def integrate_seaice_masked_data(self, masked_data, method: str, region: str):
+        """Integrate the masked data over the spatial dimension to compute sea ice metrics.
+
+        Args:
+        masked_data (xr.DataArray): The masked data to be integrated.
+        method (str): The method to compute sea ice metrics. Options are 'extent' or 'volume'.
+        region (str): The region for which the sea ice metric is computed.
+
+        Returns:
+        xr.DataArray: The computed sea ice metric.
+        """
+
+        self.logger.debug(f'Calculate cell areas for {region}')
+
+        # get info on grid area that must be reinitialised for each region
+        areacello = self.reader.grid_area
+
+        # get the region box from the region definition file
+        box = self.regions_definition[region]
+
+        # log the computation
+        self.logger.info(f'Computing sea ice {method} for {region}')
+
+        # regional selection
+        areacello = area_selection(areacello, lat=[box["latS"], box["latN"]], lon=[box["lonW"], box["lonE"]], 
+                                   loglevel=self.loglevel)
+        if method == 'extent':
+            # compute sea ice extent: exclude areas with no sea ice and sum over the spatial dimension, divide by 1e12 to convert to million km^2
+            seaice_metric = areacello.where(masked_data.notnull()).sum(skipna = True, min_count = 1, 
+                                                                       dim=self.reader.space_coord) / 1e12
+        elif method == 'volume':
+            # compute sea ice metric: exclude areas with no sea ice and sum over the spatial dimension
+            seaice_metric = (masked_data * areacello.where(masked_data.notnull())).sum(skipna = True, min_count = 1, 
```

Comment:
@tovaz92 I think is there is the problem you mention about sea ice. units here are not converted from `m^3` to `km^3` as in the extent case!